bower.json

```diff
@@ -23,6 +23,8 @@
     "string.startsWith": "*",
     "angularLocalStorage": "^0.3.2",
     "ng-walkthrough": "^1.0.0",
-    "animate.css": "^3.5.2"
+    "animate.css": "^3.5.2",
+    "nz-tour": "^1.2.1",
+    "jquery": "^3.1.0"
```

Comment:
Why do we need jquery? I've been trying to avoid using jquery because mixing and matching two different frameworks can lead to unexpected effects. It is not clear to me what functionality you are using from jquery. Is it a dependency of nz-tour?